src/pages/home/categories-list/tests/categories-list.spec.js

```diff
@@ -22,14 +24,27 @@ useQuery.mockImplementation(() => ({
   error: false
 }));
 
-describe('tests for categories list', () => {
-  const wrapper = mount(
-    <CategoriesContextProvider>
-      <CategoriesList />
-    </CategoriesContextProvider>
-  );
+describe('tests for categories list component', () => {
+  beforeEach(() => {
+    render(
+      <CategoriesContextProvider>
+        <CategoriesList />
+      </CategoriesContextProvider>
+    );
+  });
+
+  it('should exist with carousel wrapper', () => {
+    const title = screen.getByRole('heading');
+    const list = screen.getByRole('list');
```

Comment:
Look documentation for getByRole
[get](https://testing-library.com/docs/queries/byrole/)
I would advise using there level option if it is H1 . Because h6 also has a heading role.
And for the list, maybe it would be better to check a listitem?